<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkZone</class>
 <widget class="QMainWindow" name="WorkZone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Параметры механизма</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="tx">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>alpha=   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbA"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>beta=    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbB"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>gamma=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbG"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="tx_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>47</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>L=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txL">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="tx_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>47</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>a=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txa">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="tx_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>47</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>b=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txb">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="tx_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>47</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>c=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txc">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Параметры генетического алгоритма</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Количество хромосом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbChr"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Количество эпох</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbEpo"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Значения по умолчанию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>btnDefault</zorder>
   <zorder>btnCalc</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
